3p/integration.js

```diff
@@ -243,6 +243,7 @@ import {rfp} from '#ads/vendors/rfp';
 import {rnetplus} from '#ads/vendors/rnetplus';
 import {rubicon} from '#ads/vendors/rubicon';
 import {runative} from '#ads/vendors/runative';
+import {sabavision} from '#ads/vendors/sabavision';
```

Comment:
Changes in this file are no longer needed, sorry this probably changed when this PR was pending.